Cleanup unused package scripts

mattrunyon opened the pull request:
Fixes #880. Removes several unused package scripts now that we've moved to Vite.

This is a nice step before lerna v6 which uses nx since nx config includes any script that can be cached. Just helps keep things clean.

mofojed reviewed (commented):
> ● Validation Error:
> Directory /home/runner/work/web-client-ui/web-client-ui/packages/*/src in the roots[0] option was not found.

mattrunyon commented:
That error seems to be from me adding `roots` to the unit test jest config. I added that to try to quiet jest's initial spam about duplicate mocks, but I removed it since it's not a big deal
